### ISSUE TITLE
Trying to rip album from luscious.net redirects to old.luscious.net and doesn't download

### ISSUE THREAD

**Issue by keatit71** [RipMeApp/ripme#1782]
* Ripme version: 1.7.93
* Java version:  java version "1.8.0_161"
                        Java(TM) SE Runtime Environment (build 1.8.0_161-b12)
                        Java HotSpot(TM) 64-Bit Server VM (build 25.161-b12, mixed mode)

* Operating system: Windows 10 10.0.18363.1139

* Exact URL you were trying to rip when the problem occurred:  https://old.luscious.net/albums/family-moments_373027/
* Please include any additional information about how to reproduce the problem:

## Expected Behavior

Expected to download the album

## Actual Behavior

Downloading https://old.luscious.net/albums/family-moments_373027/
Rip complete, saved to D:\Porn\Other\Utility\rips\luscious_family-moments_373027
Failed to load https://old.luscious.net/albums/family-moments_373027/ after 1 attempts

**Comment by Avibharti35** [comment]
Yes after trying multiple times it starts but downloads only few pictures and then again starts giving errors.

**Comment by jak502021** [comment]
Was this ever resolved? Using v1.79.93 now on a mac and it seems nothing on the luscious site will rip.
Most will scroll in RipMe in yellow text "Already downloaded", but they've never been downloaded. Others just give that red error.
